std/format.d

```diff
@@ -1120,8 +1121,8 @@ struct FormatSpec(Char)
             {
                 if (trailing.ptr[0] == ' ')
                 {
-                    while (!r.empty && std.ascii.isWhite(r.front)) r.popFront();
-                    //r = std.algorithm.find!(not!(std.ascii.isWhite))(r);
```

Comment:
I think @jmdavis meant this for clarification, b/c it's easy to use `std.utf.isWhite` when a much cheaper ascii check is enough, and `std.ascii.isWhite` where a UTF-8 aware check is necessary.
It actually seems that this code here should be using `std.utf`.